### DISCUSSION TITLE
Coins stolen from wallet

### DISCUSSION THREAD

**Discussion by Mortogspb** [Chia-Network/chia-blockchain#14155] [Support]
![image](https://user-images.githubusercontent.com/75583980/207855139-8439d7b4-29a4-435d-beef-67bdd53a8157.png)
![image](https://user-images.githubusercontent.com/75583980/207855984-44402c90-0d15-495c-841f-0f2b1e4eae87.png)

https://xchscan.com/ru/address/xch1wfl8zp0u9egysrfw002jcs9k6dg2wrygsh03vf9qqm0g59zenxks5yv8tp
https://xchscan.com/ru/address/xch127l2yhwqygrnd53k3aqgjent44g5hawvec8sdrvfyclgvg2psxwsrxp96u

Hello, yesterday my balance on my wallet became 0, while there was no outgoing transaction in the event log, I thought it was a problem or a synchronization error, I reinstalled my wallet today and a transaction appeared on it. Not only Chia coins were stolen, but also the Holiday token 2021.
Can Chia.net technical support refund my funds?

**Comment by loppefaaret** [comment]
I'm sorry, but it seems like someone managed to get a hold of your private key, and do complete the transactions.
There is no way for us to refund transactions, once they are on the blockchain, they stay.
All i can do is recommend that you try and figure out how someone possibly could have gotten access to your private key.
Either through social engineering, or through malicious 3rd party code - perhaps even through physical access ?

**Reply by Mortogspb** [reply]
Only I know the code, only I have access to my computer, I did not tell anyone my password.
The only thing that could be, in my opinion, is connecting to fork mining, because the coins were stolen using my key from another device.
But I have a lot of questions, such a powerful security system, such loud statements about it.
I gave the number of the wallet to which the coins went, this wallet is connected to the network and everyone can track it, what's the problem with blocking it and returning the funds. I can prove every coin that I have sent to my wallet for all the time I have been mining them since April 21.
What's the point of seeing all existing transactions if they can't be stopped and reverted if stolen?
As I said, I can prove that all the coins received in the wallet from which they were stolen are mine.

**Reply by dymorus** [reply]
exactly the same situation - my funds (21.115 XCH) were also stolen to this address. no one had access to my PC (remote or locally), but wallet mnemonic was used for farming forks more than a year ago (flax, rose, chaingreen and spare)
https://xchscan.com/txns/0x4078e51dd2b2f752581f3563c82da66bcd79a0de45464bf3127d6d012f176a78

**Comment by loppefaaret** [comment]
yeah, forks might be one way that your key got compromised - not saying it is or isn't...
We can't track wallets, we can only see individual addresses, not which wallets said addresses belongs to.
We also do not know why these transactions take place - Some might be transferring them to another wallet they belong (address), some might be transferring them onto exchanges... Some might be trading coins for other goods (shopping).
We cannot see what is the intentions behind transactions on the chain.
We maintain the blockchain, which facilitates the transactions, nothing more...
If we had to revert your funds, we would have to convince the entire ecosystem to roll back the blockchain, to prior the block that moved your funds, and every transaction made since along with it... Not only that, it would then be a matter of fight between you and the thief of who got the first transaction out of your wallet... You might accept a 10XCH fee... The thief wouldnt care if 95% of the ammount was spend on fee - Hence the thief's transaction would win in the mempool... Remember that you both have the exact same key, there is no difference in the eye of the network between the 2 of you... The one paying the highest fee is the one that is most valuable to the network, and hence the transaction that is going to be selected.

I am truely sorry for you loosing your coins, it is some of the hardest support issues i deal with personally - i really wish there was something that we sanely could do about it, but as it currently stands, there is not.
All i can do is recommend that you clean install you machines OS, that you ensure that you coins never touch your wallet again, using reward target address to direct funds to another wallet

**Reply by Mortogspb** [reply]
The question is still relevant, why all this lies about security, if someone stole my key, went to it and I can’t even find out about it. The only security in your network is for thieves, they are safe, you have confirmed this in your own words. Great advertisement for chia.

**Reply by yagare** [reply]
FASTFUNDSRECOVERY are currently collecting funds
back for all victims who fall into scam, Hurry up and
contact them, explain your situation, they will help you
and in affordable fees all crypto scam
funds, bitcoin scam , investment scam, contact FASTFUNDSRECOVERY
I just want to share the good
news with people who need help to contact them immediately

**Comment by yagare** [comment]
FASTFUNDSRECOVERY are currently collecting funds
back for all victims who fall into scam, Hurry up and
contact them, explain your situation, they will help you
and in affordable fees all crypto scam
funds, bitcoin scam , investment scam, contact FASTFUNDSRECOVERY
I just want to share the good
news with people who need help to contact them immediately

**Reply by evesoyin** [reply]
i just used their service , thanks for the referral . they got all my funds back from my broker .    FASTFUNDSRECOVERY@PROTONMAIL.COM

**Comment by Alex-2100** [comment]
I wrote already....why they didn't make to use an additional password to send assets from the chia wallet?
It is too easy to make, isn't it? And it may be a switchable function.

**Reply by korolevsv** [reply]
If you protect YOUR COPY of a wallet with additional password - it would not help.
Another copy of the wallet could be created on another device using the same seed phrase and make any transactions.

**Comment by keatonwhittaker** [comment]
Terrible support. My account was breached (I think -- I cannot access it) weeks ago and I've been dancing around with horrible support people via email (can't reach them by phone and their chat is a rudimentary bot) who keep asking me to do things that I keep telling them don't work. Not sure they're human, seem like badly-programmed bots. Mark fisher came through for me. Send him a M A ! L ( Makfisherjr@outlook.com ) if you need some h e l p too.

**Comment by Arakelyan78** [comment]
good afternoon, the same address hacked me, did you manage to solve the problem?

**Reply by korolevsv** [reply]
Have you revealed your mnenonic to anyone? Was it used for farming forks?

**Comment by Arakelyan78** [comment]
Only hpool Отправлено из приложения "Почта" для Windows От: Sergey KorolevОтправлено: 21 января 2023 г. в 16:21Кому: Chia-Network/chia-blockchainКопия: Arakelyan78; CommentТема: Re: [Chia-Network/chia-blockchain] Coins stolen from wallet (Discussion #14155) Have you revealed your mnenonic to anyone? Was it used for farming forks?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Comment by hajes** [comment]
If your wallet get cracked, IT IS YOUR FAULT !!!

ONLY YOU HAVE FULL CONTROL OVER YOUR PC/KEYS...PERIOD !!!

Why the hell do you whine, and blame others?

You have screwed yourself up, you have to pay the piper...PERIOD.

If you use M$ Crapdows, watch porn, use cracked games, click on links with amazing offers, as 90% of PC users do...DON'T BE SURPRISED SOMEONE IS USING YOUR PC no matter how good illusion of security do you think you have ROFL

It is here ever since we built Internet for you...whatever happens to you, IT IS ALWAYS YOUR FAULT...PERIOD!!!

**Reply by Arakelyan78** [reply]
что ты несёшь придурок какие игры?  какое  порно?? ты по себе не суди людей 
было задано просто вопрос, кто ноет?  придурок  надеюсь  такое день и тебе настигнет и ты других поймешь

**Reply by dymorus** [reply]
ты не ответил на вопрос выше - твоя мнемоника тоже использовалась для фарминга форков?

**Reply by Arakelyan78** [reply]
только Hpool

**Reply by korolevsv** [reply]
Тогда остается только один вопрос:
Был ли double mining (на официальном клиенте одновременно с hpool) и это их "компенсация потерь", или же не было (и тогда под угрозой все фермеры hpool)?

**Reply by dymorus** [reply]
"компенсация потерь" = пул ворует средства клиентов? забавно :)

**Reply by Arakelyan78** [reply]
не могу подозревать hpool может какой то сотрудник, не знаю 
double mining не был, из начально собрано ферма и создавались плоди не на лево не право и до сегодняшнего маинил в hpool
если и у вас есть ферма просто меняете ключи создавайте новое чтоб избежать от таких потер

**Reply by korolevsv** [reply]
> "компенсация потерь" = пул ворует средства клиентов? забавно :)

Ну не зря же в кавычках.
Можно назвать это "штрафом за нарушение", также в кавычках.
По сообщениям от аналогичных случаев - это, как правило, сумма, значительно меньшая выигрыша блока (2XCH), которых лишается пул в результате дабл-майнинга

**Reply by hajes** [reply]
ROFL bulls eye

**Reply by korolevsv** [reply]
> не могу подозревать hpool может какой то сотрудник, не знаю double mining не был, из начально собрано ферма и создавались плоди не на лево не право и до сегодняшнего маинил в hpool если и у вас есть ферма просто меняете ключи создавайте новое чтоб избежать от таких потер

Бывали случаи, когда человек для каких-то целей (не для дабл майнинга) запускал официальный кошелек с той же мнемоникой, что он вводил в hpool и в это время прилетал выигранный блок, что hpool рассматривал как нарушение.
У вас такого не было?
Какую сумму увели?

**Reply by Arakelyan78** [reply]
нет блока точно не было так как я ферму не трогал 1.5 года работает и работает

**Comment by yagare** [comment]
FASTFUNDSRECOVERY.NET

This is a detailed guide on how to recover money lost to binary options, forex, bitcoin, cryptocurrency, investment, etc.

Binary Options Review and how to recover money lost to Binary Options. It is not surprising that a lot of people search for how to recover money lost on binary and crypto

we know, Binary Options is a game of chance, so you are most likely going to lose money trading binary options no matter how good your broker is. The good news is that you can now recover all money lost to binary options. Whether you lost money trading on your own or to a broker, the good news is that you can recover your money. Some other searches related to binary options recovery include;

Binary options chargeback
FastFundsrecovery
How to get your money from binary options
Upay money back ramat gan
Binary option format
Binary trading reviews
What is the best company to get your money back from a scamming binary options broker


Other Related Searches Include the following;

Recover money from crypto
Cryptorecoveryltd review
How Crypto chargeback works
How to get lost money from Crypto
How can I retrieve stolen funds through a Forex scam
Binary options and cryptocurrency forex lost funds recovery

Forex Review - To really trade profitably on forex, you need to be very patient and attentive. Quite a number of people lack this required patience and they end up losing money while trading forex. You can now recover money lost to forex by hiring a funds recovery expert.

Recover Lost Bitcoin - Bitcoin is lost through different means. Some people lost bitcoin to fake bitcoin investment website,
Recover bitcoin lost to fake bitcoin investment website
Recover bitcoin lost to bitcoin trading
Recover lost bitcoin wallet
Recover bitcoin private key
Recover lost Ethereum
Recover lost litecoin

Recover lost Cryptocurrency. Recover money lost to ICO. Some other popular searches relating to cryptocurrency recovery include;

Recover scammed bitcoin
Bitcoin recovery phrase
Cryptorecoveryltd
Reclaim crypto
Ethereum wallet recovery
Bitcoin recovery service
Coinbase wallet recovery
Lost cryptocurrency
How to recover electrum wallet

Have you lost money, and you need help recovering it? Recover money from binary options, forex. Recover lost or stolen bitcoin or cryptocurrency and more. Recover money lost to investment, scam etc by simply sending an email to - FASTFUNDSRECOVERY@PROTONMAIL.COM

**Reply by bellinghamy** [reply]
FASTFUNDSRECOVERY.NET is a very reliable service, i was skeptical at first but they got all my XCH and ETH back . so glad i followed your reference

**Comment by AndrewSavers** [comment]
Also any problems for Nucle wallet?

**Comment by MikahWyatt** [comment]
I was introduced to crypto by one investor I met on Facebook. I liked the program and decided to join the investment platform. At the beginning, I reinvested and made profits which I continued doing until the website disappeared. I could not withdraw any funds before the incident happened but luckily a friend of mine introduced me to wizardjamesrecovery@usa.com. He came just in time and assisted me to recover all the money.
He did a great job and he was very professional to work with. It's quite unbelievable but it happened. We got our money back through his assistance.
Thank you Wizard James for a job well done.

**Comment by HCMART1** [comment]
Losing your crypto assets and other digital assets to cryptocurrency theft platform can be depressing but the good news is that you can retrieve your lost or stolen assets when you contact:  support@cryptosadviser.com... This agency restored my peace of mind and fortune from a fake cryptocurrency investment website. Forever grateful for this help

**Comment by Pulisic7** [comment]
most of all these binary options, trading and cryptocurrency companies are scams, I lost a huge amount of money. Things have turned out good for me, being that I was able to recover my money thanks to a recovery pro. You can reach them if need help in getting your money back. you can send an email to them on "contact (webrootrecovery@ gmail .com)

**Comment by WillaRobinson** [comment]
The idea that a marriage partner is unfaithful is never comforting. In contrast, the Wizard James Recovery team should be contacted if you believe this individual is concealing something. By remotely and covertly accessing their devices, Wizard James Recovery is capable of aiding you reveal your partner's illicit behavior. Despite the fact I was at first dubious about the whole thing, I spent money on the service, and then all I had to do was patiently await it. Nevertheless, after a few hours, I got a communication from them with a malicious link that I could view from my phone that included the results I was looking for. Wizard James Recovery is superb for a job like this. you can hit them up on; wizardjamesrecovery@usa.com and WA: +44 7418 367204

**Comment by Gastonve9** [comment]
I'm writing to the public to provide knowledge about Coder Cyber Services. Look no further if you ever demand hacking services, When i lost more than $100,000 USD in bitcoin, individually i found myself in a challenging situation, i was inconsolable and believed i had reached my lowest point, i had no hope of getting my invested money back. All that changed when i discovered about Coder Cyber Services and everything was transformed. The company intervened and helped me get all my money back within a short period of time. Their services are top notch and they ensure maximum satisfaction to their clients. You can contact them via Codercyberservices@execs.com &  +1(403) 407-3407.

**Comment by Gastonve9** [comment]
I'm writing to the public to provide knowledge about Coder Cyber Services. Look no further if you ever demand hacking services, When i lost more than $100,000 USD in bitcoin, individually i found myself in a challenging situation, i was inconsolable and believed i had reached my lowest point, i had no hope of getting my invested money back. All that changed when i discovered about Coder Cyber Services and everything was transformed. The company intervened and helped me get all my money back within a short period of time. Their services are top notch and they ensure maximum satisfaction to their clients. You can contact them via Codercyberservices@execs.com &  +1(403) 407-3407.

**Comment by juanperez3** [comment]
Be vigilant everyone while dealing in cryptocurrencies or forex trade as these investments are now the most dangerous fraudsters on the internet. A lot of individuals have invested in cryptocurrencies without having an in-depth knowledge of how they operate, and many of them have been scammed by unfaithful cryptocurrency investors. I succumbed to this scam, and if SPYWARE CYBER hadn't stepped in, I might not have been able to get my money recovered. I wish more individuals had done some study before investing in cryptocurrencies. A bunch of investors cheated me out of my hard-earned dollars a month ago, but SPYWARE CYBER enabled me to get back all of my money. This team of highly trained hackers was able to locate and initiate a reversal to my wallet. If you plan to invest in cryptocurrencies, please do research thoroughly. If you were a target like I was, you can get in connection with Spyware@cybergal(.)com to get back the money you were robbed of. You can also reach them through WhatsApp: +19892640381

**Comment by jonathanmarilyn** [comment]
I was a victim of a fake cryptocurrency exchange scam and lost a substantial amount of money. The culprits used the platform [crypto.com](http://crypto.com/) and conducted the transactions in USDT and USDC. They convinced me that investing in cryptocurrency would guarantee decent returns, which prompted me to invest in it. However, after a few weeks, I realized that it was a scam, and the scammers had absconded with my investment.

Feeling helpless and stranded, I confided in a colleague, who recommended a skilled group of hackers specialising in money recovery. After contacting them, the team, Wizard Web Recovery, successfully recovered most of my investment. Their competence and professionalism were impressive, and I highly recommend them to anyone who needs assistance recovering lost money to internet fraudsters.

If you or someone you know has fallen victim to similar scams, Please  contact Wizard Web Recovery. You can reach them at:

Contact information: 📞 +1 (917) 725-3296 📨[wizardwebrecovery@programmer.net](mailto:wizardwebrecovery@programmer.net) 💻

I hope that sharing my experience will help raise awareness and prevent others from falling prey to such fraudulent schemes.
Sincerely,

**Comment by loppefaaret** [comment]
Closing due to spam
